### DISCUSSION TITLE
Where to discuss (linux)brew casually and get friendly help?

### DISCUSSION THREAD

**Discussion by jacktose** [Homebrew/discussions#379] [Everyday usage] [answered]
Is this (Homebrew/discussions) the appropriate place for inexpert homebrew/linuxbrew users to ask questions, offer suggestions, get help with their brew installs, etc.?

I have been a pretty dedicated user of homebrew and linuxbrew, advocated for it at work, contributed a little feedback, code, and money. Please understand that I am asking in earnest and not trying to waste anyone's time.
In the past, I dropped into Discourse (and wasn't there a gitter or discord or something?) to ask noob questions, get help with weird issues, etc. I generally got earnest and friendly responses, if not always timely or authoritative, which seems fine for the casual discussion side of a big project.

Now the Discourse is deprecated in favor of GitHub Discussions, so I asked my latest question there. I got a bit of explanation, but mostly I was made fun of and rebuked for complaining and “demotivating“ the maintainers, then locked out.
Now, my ego will survive, but I will want again to ask about stuff that doesn't rise to the level of an Issue, and if there's a way or place to do that that's preferred by the folks close to the project, I'd like to abide by their wishes. Moreover, I expect there are other users looking for some level of involvement between silently consuming the product and actively contributing.
I suggest that it behooves the project to have a friendly and casual venue, to encourage these users and potential contributors. If that's Discussions, I think more consideration should be given to its tone and guidelines; in particular, the expectations should be distinguished from those in Issues and Pull Requests. If it's somewhere else (reddit? discord? a hashtag?), please point me there and I'll leave this section alone.

Thanks!

**Comment by iMichka** [comment, marked as answer]
In my opinion this discussion board is a little bit user-oriented, and anything can be discussed here as long as it is respectful.
I think you are referring to this discussion https://github.com/Homebrew/discussions/discussions/195, and in my opinion it was the right place to open it.

I personally don't read or answer much on this discussion board as a maintainer, I just watch it to see if bigger issues pop up, but most of the times it's discussions about stuff that is already documented somewhere or where anyone can open a small pull request to fix thing.

I think we did not yet strike the right balance between the GitHub issues (which is the official place to report bugs that are reproducible), and this place where design choices, weird behaviours (but expected ones) can be discussed.

**Reply by Yakubu964** [reply]
That's great

**Reply by jacktose** [reply]
Thanks!
Maintaining software is hard, and maintaining communities is different-hard. I want Homebrew to be successful at both.
